rcmd/install.go

```diff
@@ -498,9 +513,51 @@ func InstallPackagePlan(
 			log.Errorf("did not install %s", pkg)
 		}
 	}
+
 	if anyFailed {
 		log.Errorf("installation failed for packages: %s", strings.Join(failedPkgs, ", "))
 		return fmt.Errorf("failed installation for packages: %s", strings.Join(failedPkgs, ", "))
 	}
 	return nil
 }
+
+func writeDescriptionInfo(ir InstallRequest) {
+	updateDescriptionInfo(
+		filepath.Join(ir.InstallArgs.Library, ir.Package, "DESCRIPTION"),
+		ir.ExecSettings.PkgrVersion,
+		ir.Metadata.Metadata.Config.Type.String(),
+		ir.Metadata.Metadata.Config.Repo.URL,
+		ir.Metadata.Metadata.Config.Repo.Name,
+		true)
+}
+
+func updateDescriptionInfo(filename, version, installType, repoURL, repo string, writeFile bool) ([]byte, error) {
+	var update []byte
+	fs := afero.NewOsFs()
+	dcf, err := afero.ReadFile(fs, filename)
+	if err == nil {
```

Comment:
this error is never handled, other than returning it, but in writeDescription you're not catching the error and doing anything about it